profiles/desc/curl_default_ssl.desc

```diff
@@ -0,0 +1,11 @@
+# Copyright 1999-2020 Gentoo Foundation.
+# Distributed under the terms of the GNU General Public License v2
+
+# This file contains descriptions of CURL_DEFAULT_SSL USE_EXPAND flags for net-misc/curl
+
+gnutls - Use GnuTLS
+libressl - Use LibreSSL
+mbedtls - Use mbed TLS
+nss - Use Mozilla's Network Security Services
+openssl - Use OpenSSL
+winssl - Use WinSSL (only with elibc_Winnt)
```

Comment:
Please fix your editor to leave trailing newlines.

Reply:
I think that was actually me doing it manually since I was confused by why nss and winssl had them in the first place. I will avoid doing so in the future. For the record do you know the reason they are present?

Reply:
Try `cat file` on a file without trailing newline.

Reply:
Oof. Yeah. Will fix. I usually see in commits that I add newlines to the end of files that don't have them rather than remove them so this case is egregiously bad, will see what is going on with it.